lib/synced/strategies/full.rb

```diff
@@ -192,6 +192,15 @@ def search_params
         end]
       end
 
+      def scope_params
+        case @scope
+        when Rental
```

Comment:
`Synced::Model` should define the method with default value and included module would oveerwrite it. That way the models will always quack properly.